form/create_custom_field_type.rst

```diff
@@ -67,14 +67,31 @@ check out the `ChoiceType`_ class.
     extend from ``AbstractType`` is only a convenience way of implementing the
     required ``FormTypeInterface``.
 
-There are three methods that are particularly important:
+There are a few methods that are particularly important:
 
 .. _form-type-methods-explanation:
 
+``getParent()``
+    This return the class name of the type configuring the form before. It
+    means all the other methods below will be called with this parent type
+    and all its type extensions before calling the ones defined in your
+    custom type.
+    By default, all classes inherit the ``AbstractType::getParent()`` which
```

Comment:
"[...] all classes extend the `AbstractType` class [...]" or "[...] all classes inherit the `AbstractType::getParent()` method [...]"?